src/redux/todo/reducer.js

```diff
@@ -8,7 +8,7 @@ function* createIds() {
 }
 const generateId = createIds();
 
-export default (state = INITIAL_STATE, action) => {
+export default function reducer(state = INITIAL_STATE, action) {
```

Comment:
Why the change? Use the arrow function instead of using the functon keyword.

Reply:
Done on another branch called "bookList".